### PR TITLE
docs: Blog post for 0.19 release. #0.19

### PR DESCRIPTION
**Pull request by mturoci**

### CODE REVIEW

File: website/blog/2021-11-05-release.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    ],
+    theme='my-awesome-theme'
+)
+```
+
+:::tip
+You can register multiple themes, which might be handy for switching between the light / dark mode of your app.
+:::
+
+## Theme generator
```
Comment (mtanco) [inline website/blog/2021-11-05-release.md:41]: Consider adding a line asking people to post their favorite custom themes to the Discussions

File: website/blog/2021-11-05-release.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+---
+
+With the latest release, we are excited to announce Wave becoming more colorful! In addition to existing color themes (`default`, `neon`) we added `h2o-dark` and plan on expanding the list to at least a dozen
+so stay tuned. If interested, you can track the progress [here](https://github.com/h2oai/wave/issues/696). However, having only predefined color palettes is limiting so we decided to allow a custom, user-defined palettes.
+
+<!--truncate-->
+
+## How it works?
+
+Firstly, a new `ui.theme` needs to be registered within the `themes` attribute in `ui.meta_card`. It takes the name of the theme and 4 main colors as arguments.
```
Comment (mtanco) [inline website/blog/2021-11-05-release.md:18]: Consider explaining explicitly what the 4 main colors are, what's the right one to use for a fun accent, which should be more solid colors?

File: website/blog/2021-11-05-release.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+./venv/bin/wave run examples.theme_generator
+```
+
+Then visit <http://localhost:10101/demo>.
+
+![theme-generator](assets/2021-11-05/theme-generator.gif)
+
+This Wave app provides:
+
+* Immediate visual feedback.
```
Comment (mtanco) [inline website/blog/2021-11-05-release.md:55]: I would move this is the top of this sub heading

File: website/blog/2021-11-05-release.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+* Color contrast warnings when a particular contrast ratio is less than [4.5](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).
+* Quick theme copy-pasting.
+
+## Can individual colors be changed?
+
+In general, the answer is no. Wave tries to be smart and do all the minor decisions for you, e.g. picking a color for primary/secondary buttons, checkboxes, etc. However, some cards allow changing colors of particular parts, for example, icons, but this is more of an exception rather than a strict rule.
```
Comment (mtanco) [inline website/blog/2021-11-05-release.md:61]: Add a sentence on why we do this, by grouping colors and handling it we let you have good ux and focus on your product - or something like taht

File: website/blog/2021-11-05-release.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+If full control over colors/typography etc. is needed, going with regular HTML/CSS/JS is probably a better choice.
+
+## Bunch of new cards added
```
Comment (mtanco) [inline website/blog/2021-11-05-release.md:65]: Suggestion to add a link to the example for each new card

**Review by mtanco — COMMENTED** [review]
Looks good! Added some thoughts and suggestions

File: website/blog/2021-11-05-release.md (hunk 1 above)
Comment (mturoci) [reply website/blog/2021-11-05-release.md:41]: Great idea, done!

File: website/blog/2021-11-05-release.md (hunk 2 above)
Comment (mturoci) [reply website/blog/2021-11-05-release.md:18]: Added a description for each color, but not sure what is meant by the "fun accent". The general theme impression is based on colors altogether or am I missing something?

File: website/blog/2021-11-05-release.md (hunk 3 above)
Comment (mturoci) [reply website/blog/2021-11-05-release.md]: Agree, thanks.

File: website/blog/2021-11-05-release.md (hunk 4 above)
Comment (mturoci) [reply website/blog/2021-11-05-release.md]: Good point, thanks!

File: website/blog/2021-11-05-release.md (hunk 5 above)
Comment (mturoci) [reply website/blog/2021-11-05-release.md]: Agree, thanks!

**Review by lo5 — APPROVED** [review]
Thanks @mturoci!

**Pull request merged**